app/javascript/src/redux/welcome/fetchAPi.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const url = `http://127.0.0.1:3000/api/v1/greetings/index`;
```

Comment:
- [ ] I can't find the above route `api/v1/...` in controllers to fetch the greetings. Please make sure to check it and fix it.